### PR TITLE
Adds Highjack, a new drug for IPC's!

### PR DESCRIPTION
**Pull request by Pontenerd**
## About The Pull Request

Adds a new drug, similar to space drugs, but only for ipcs, when overdosed, they detach their heads.

## Why It's Good For The Game

This now means all our crewmembers can be arrested for being high at their station

## Changelog
:cl:
add: Added Highjack, a drug similar to space drugs, but for IPC's
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 1]
```diff
@@ -32,6 +33,27 @@
...
+	if(prob(7))
+		M.emote(pick("buzz","beep","ping","buzz2"))
+	..()
+
+/datum/reagent/drug/highjack/overdose_process(mob/living/M)
+	var/obj/item/bodypart/head/head = M.get_bodypart("head")
+	if(prob(30))
+		to_chat(M, "<span class='userdanger'>You feel really lightheaded...</span>")
+		head.dismember()
+
```
Comment (Bokkiewokkie) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:56]: Put this in the NSV folders

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 2]
```diff
@@ -4,6 +4,12 @@
 	results = list(/datum/reagent/drug/space_drugs = 3)
 	required_reagents = list(/datum/reagent/mercury = 1, /datum/reagent/consumable/sugar = 1, /datum/reagent/lithium = 1)
 
+/datum/chemical_reaction/highjack
+	name = "Highjack"
+	id = /datum/reagent/drug/highjack
+	results = list(/datum/reagent/drug/highjack = 3)
+	required_reagents = list(/datum/reagent/medicine/liquid_solder = 1, /datum/reagent/medicine/system_cleaner = 1, /datum/reagent/medicine/radioactive_disinfectant = 1,/datum/reagent/uranium/radium = 1 )
+
```
Comment (Bokkiewokkie) [inline code/modules/reagents/chemistry/recipes/drugs.dm:12]: Ditto

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 3]
```diff
@@ -33,6 +34,13 @@
 		M.hallucination += 5
 	..()
 
+/datum/reagent/drug/highjack/overdose_process(mob/living/M)
+	var/obj/item/bodypart/head/head = M.get_bodypart("head")
+	if(prob(30))
+		to_chat(M, "<span class='userdanger'>You feel really lightheaded...</span>")
+		head.dismember()
+
+
```
Comment (Bokkiewokkie) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:43]: Please move these into a NSV file as well (and remove that added first line in the file.

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 4]
```diff
@@ -4,6 +4,7 @@
 	results = list(/datum/reagent/drug/space_drugs = 3)
 	required_reagents = list(/datum/reagent/mercury = 1, /datum/reagent/consumable/sugar = 1, /datum/reagent/lithium = 1)
 
+
```
Comment (Bokkiewokkie) [inline code/modules/reagents/chemistry/recipes/drugs.dm:7]: extra line :taco:

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 5]
```diff
@@ -1,3 +1,4 @@
...
```
Comment (Bokkiewokkie) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:1]:
```suggestion
```

File: nsv13/code/modules/reagents/chemistry/recipes/drugs.dm [hunk 6]
```diff
@@ -0,0 +1,5 @@
+/datum/chemical_reaction/highjack
+	name = "Highjack"
+	id = /datum/reagent/drug/highjack
+	results = list(/datum/reagent/drug/highjack = 3)
+	required_reagents = list(/datum/reagent/medicine/liquid_solder = 1, /datum/reagent/medicine/system_cleaner = 1, /datum/reagent/medicine/radioactive_disinfectant = 1,/datum/reagent/uranium/radium = 1 )
```
Comment (covertcorvid) [inline nsv13/code/modules/reagents/chemistry/recipes/drugs.dm:5]:
```suggestion
	required_reagents = list(/datum/reagent/medicine/liquid_solder = 1, /datum/reagent/medicine/system_cleaner = 1, /datum/reagent/medicine/radioactive_disinfectant = 1, /datum/reagent/uranium/radium = 1)
```

**Review by covertcorvid — APPROVED** [review]

**Review by covertcorvid — APPROVED** [review]

**Pull request merged**